### PR TITLE
Proofreading of previous PRs and other minor changes.

### PR DESCRIPTION
**Pull request by abravosuse**
### PR creator: Description

Proof reading of previous PRs and other minor changes.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -161,8 +161,8 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         <listitem>
           <formalpara>
             <title>Delivery mechanism:</title>
-            <para> A set of container images from the &suse;
-              registry together with a Helm chart for easy
+            <para> A set of container images from the public &suse;
+              registry together with a Helm chart that facilitates their
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:165]:
```suggestion
            <para> A set of container images from the &suse; public
              registry together with a Helm chart that facilitates their
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -180,14 +180,13 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
-                &rancher.k8s.engine; (RKE) version 1 or
+                support, &suse; recommends &suse; &rancher.k8s.engine; (RKE) version 1 or
                 2. </para>
             </listitem>
             <listitem>
-              <para> If you do not have a &k8s; enterprise solution and you would
-                like to test &trentopremium;, &suse; Rancher's K3s could be an easy
-                way to get started. Keep in mind that K3s default installation process 
+              <para> If you do not have a &k8s; enterprise solution but you sill would
+                like to try &trentopremium;, &suse; Rancher's K3s provides you with an easy
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:188]:
```suggestion
              <para> If you do not have a &k8s; enterprise solution but you still want
                to try &trentopremium;, &suse; Rancher's K3s provides you with an easy
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -335,7 +334,7 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         </step>
         <step>
         <para>
-          Install the &t.server; by pulling the Trento Helm chart with Helm:
+          User Helm to install &t.server; using the Trento Helm chart:
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:337]:
```suggestion
          Use Helm to install &t.server; with the Trento Helm chart:
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -403,8 +402,7 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
       </step>
       <step xml:id="st-deploy-k3s">
         <para>
-          With the same user that installed K3s, install &t.server; by pulling the Helm chart
-          using Helm:
+          With the same user that installed K3s, use Helm to install &t.server; using the Helm chart:
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:405]:
```suggestion
          With the same user that installed K3s, use Helm to install &t.server; with the Helm chart:
```

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -2186,6 +2185,11 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
         </screen>
    <para>A few things to consider:</para>
       <itemizedlist>
+                <listitem>
+          <para>Before updating &t.server; ensure that all the &t.agent;s in the environment are supported by the target version.
+           See ssection <xref linkend="sec-trento-compatibility-matrix" /> for details.
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2190]:
```suggestion
           See section <xref linkend="sec-trento-compatibility-matrix" /> for details. 
```

File: trento/xml/article_sap_trento.xml [hunk 6]
```diff
@@ -2433,10 +2437,7 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
             <screen>&prompt.user;<command>chmod</command> +x dump_scenario_from_k8.sh</screen>
           </step>
           <step>
-            <para>Set the environment variable <envar>KUBECONFIG</envar> appropriately.</para>
-          </step>
-          <step>
-            <para>Generate the dump:</para>
+            <para>Ensure <command>kubectl</command> is connected to the K8s cluster where &t.server; is running and exceute the script as follows:</para>
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:2440]:
```suggestion
            <para>Ensure <command>kubectl</command> is connected to the K8s cluster where &t.server; is running and execute the script as follows:</para>
```

**Review by abravosuse — COMMENTED** [review]
Minor corrections over original changes

File: trento/xml/article_sap_trento.xml [hunk 7]
```diff
@@ -2328,7 +2332,7 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
     <para>
       Any &suse; customer, with a registered &sles4sap;&nbsp;15 (SP1 or higher) distribution,
      experiencing an issue with &trentopremium;, can report the problem either directly in the &scc;
-     or through the corresponding vendor, depending on their licensing model. Any how, the problem has to reported under
+     or through the corresponding vendor, depending on their licensing model. Anyhow, the problem has to reported under
      &sles4sap;&nbsp;15 and component trento.
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2336]: I think, here we want something from the user (to file a bug report), so I would address the user with "please" and make it sound not too harsh:

```suggestion
     or through the corresponding vendor, depending on their licensing model. Please report the problem under &sles4sap;&nbsp;15 and component trento.
```

File: trento/xml/article_sap_trento.xml [hunk 8]
```diff
@@ -2445,6 +2446,39 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
           </step>
         </procedure>
       </section>
+
+    <section>
+      <title>Poda descriptiona and logs</title>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2451]: That sounds strange. Do you mean "pods descriptions and logs"?

```suggestion
      <title>Pods descriptions and logs</title>
```

File: trento/xml/article_sap_trento.xml [hunk 9]
```diff
@@ -2445,6 +2446,39 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+      <para> In additon to the output of the Trento Support Plugin and the Dump Scenario script, the descriptions and logs of the
+       pods of the &t.server; can also be useful for analysis and troubleshooting purposes. These descriptions and logs can be 
+       easily retrieved with the <command>kubectl</command> command, for which you need a host where command <command>kubectl</command>
+       is installed and connected to the K8s cluster where Trento Server is running.
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2455]: * I would split the sentence
* Use `&t.server;` entity
* "easily": would drop that. Easy/difficult categories are quite subjective and we don't want that our readers feel dump if they perceive something as not so easy.
* The last sentence tries to put too much information into one sentence.

If I take all this into considerations, I'll end up with this:

```suggestion
       pods of the &t.server; can also be useful for analysis and troubleshooting purposes. These descriptions and logs can be 
       retrieved with the <command>kubectl</command> command. You need a host where the <command>kubectl</command> command
       is installed and connected to the K8s cluster. The K8s cluster has to run the &t.server;.
```

File: trento/xml/article_sap_trento.xml [hunk 10]
```diff
@@ -2445,6 +2446,39 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+      <procedure>
+        <step>
+          <para>List the pods running in &k8s; cluster and their status. Trento Server currently consists of the following six pods: </para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2458]:
```suggestion
          <para>List the pods running in &k8s; cluster and their status. &t.server; currently consists of the following six pods: </para>
```

File: trento/xml/article_sap_trento.xml [hunk 11]
```diff
@@ -2445,6 +2446,39 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
         </procedure>
       </section>
+
+    <section>
+      <title>Poda descriptiona and logs</title>
+      <para> In additon to the output of the Trento Support Plugin and the Dump Scenario script, the descriptions and logs of the
+       pods of the &t.server; can also be useful for analysis and troubleshooting purposes. These descriptions and logs can be 
+       easily retrieved with the <command>kubectl</command> command, for which you need a host where command <command>kubectl</command>
+       is installed and connected to the K8s cluster where Trento Server is running.
+      <procedure>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2456]: I think, we miss a paragraph. In the last paragraph you describe it in a more general way. To lead the user to the procedure, I'd suggest:

```suggestion
      <para>To retrieve the logs for further introspection, do the following:
      </para>
      <procedure>
```

File: trento/xml/article_sap_trento.xml [hunk 12]
```diff
@@ -2445,6 +2446,39 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
...
+        <step>
+          <para>Retrieve the description of one particular pod as follows:</para>
+          <screen>&prompt.user;<command>kubectl describe pod</command> <replaceable>POD_NAME</replaceable></screen>
+        </step>
+        <step>
+          <para>Retrieve the log of one particular pod as follows:</para>
+          <screen>&prompt.user;<command>kubectl logs</command> <replaceable>POD_NAME</replaceable></screen>
+        </step>
+        <step>
+          <para>Monitor log of one particular pod as follows:</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:2476]:
```suggestion
          <para>Monitor the log of one particular pod as follows:</para>
```

**Review by tomschr — CHANGES_REQUESTED** [review]
@abravosuse Thanks for the contribution! :+1: Just some minor issues.

File: trento/xml/article_sap_trento.xml (hunk 9 above)
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:2454]: I'm fine with splitting the last sentence in two parts, but maybe three is to much. I would go for:

These descriptions and logs can be retrieved with the <command>kubectl</command> command. You need a host where the <command>kubectl</command> command is installed and connected to the K8s cluster where &t.server; is running.

After all, the sentence "You need a host where the <command>kubectl</command> command is installed and connected to the K8s cluster where &t.server; is running." is already used in the conditions to use the Trento Support Plugin and the Dump Scenario scrip.

File: trento/xml/article_sap_trento.xml (hunk 11 above)
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:2455]: Good idea adding the leading paragraph. To align better with the section goal I would write:

<para>To retrieve the descriptions and the logs for further introspection, do the following:
      </para>

Comment (tomschr) [reply trento/xml/article_sap_trento.xml:2455]: Yes, use your suggestion. :+1:

**Pull request merged**